### PR TITLE
ta Android.mk files: add LOCAL_PATH for each file

### PR DESCRIPTION
**Pull request by liuyq**
so that we can have LOCAL_SRC_FILES defined with LOCAL_MODULE,
and possible to create the dependency for LOCAL_SRC_FILES
in the device configuration repository

Signed-off-by: Yongqin Liu <yongqin.liu@linaro.org>

### CODE REVIEW

File: ta/storage/Android.mk [hunk 1]
```diff
@@ -1,6 +1,7 @@
+LOCAL_PATH := $(call my-dir)
+
 include $(CLEAR_VARS)
 LOCAL_MODULE := b689f2a7-8adf-477a-9f9932e90c0ad0a2.ta
```
Comment (jbech-linaro) [inline ta/storage/Android.mk]: Shouldn't it be `+LOCAL_SRC_FILES := $(LOCAL_MODULE)` below here also?

Comment (liuyq) [inline ta/storage/Android.mk]: Yes, should have that here as well.
Will send a new request later.

Comment (jbech-linaro) [inline ta/storage/Android.mk]: You're aware of that you can push push new changes to the same branch to update it? I.e., you don't have to close a pull request and create another one just to make changes.

Comment (liuyq) [inline ta/storage/Android.mk]: Ah, sorry, I did not know how to do that with git hub:(

**Pull request closed without merging**